File: Programação Funcional/Básicos/Scala/02.scala
object Main {
  // Função mult3 que retorna verdadeiro se o número for múltiplo de 3
  def mult3(x: Int): Boolean = x % 3 == 0

  // Função principal para testar
  def main(args: Array[String]): Unit = {
    println(mult3(9))  // Deve retornar true
    println(mult3(10)) // Deve retornar false
  }
}

File: Programação Funcional/Básicos/Haskell/02.hs
{- 
	Faça uma função mult3 x que retorne True caso a entrada seja múltiplo de 3 e False caso contrário.
-}
mult3 :: Int -> Bool
mult3 x = x `mod` 3 == 0

-- Função principal para testar
main :: IO ()
main = do
    print (mult3 9)  -- Deve retornar True
    print (mult3 10) -- Deve retornar False